src/reducers/city.reducer.js

```diff
@@ -0,0 +1,13 @@
+import { cityConstants} from '../constants';
+
+export function city(state={},action){
+
+    switch (action.type){
+        case cityConstants.ADD:
```

Comment:
You have 3 actions that do the same thing. Please leave only one.

Reply:
Those actions return the same value,but they don't the same thing.